backend/capellacollab/sessions/hooks/jupyter.py

```diff
@@ -81,11 +81,11 @@ def post_session_creation_hook(
         **kwargs,
     ):
         assert self._jupyter_public_uri.hostname
-        operator.create_public_route(
-            session_id=session_id,
+        operator.create_ingress(
+            id=session_id,
```

Comment:
I would either use `id` or `name` in both the create and delete ingress function and not two different names for the same purpose.

Reply:
Makes sense, `id` as variable name is bad practice anyway.